### DISCUSSION TITLE
Execution of Serverless Functions are all errored

### DISCUSSION THREAD

**Discussion by carldegs** [vercel/community#293] [Help] [answered]
Hi everyone! I just recently looked at the usage tab of my team and saw that from two days ago, almost all serverless function executions result in errored responses. But checking the website and looking at the real-time function logs, there seems to be no issue. All requests from the client succeeded and no error logs appear on the functions. Is this just a bug or is there something that I haven't considered that's causing this issue. Also, all invocations succeeded so not sure if it's possible. Thanks!

![image](https://user-images.githubusercontent.com/14056264/152209458-da230b16-e6db-4fe0-aa6b-de45ecd14098.png)

**Comment by Vilijan** [comment, marked as answer]
I experience the same behavior.  Check issue #289

**Reply by leerob** [reply]
Going to mark this as a duplicate - let's continue the conversation there! Thank you.
